### ISSUE TITLE
[Bug] Content is placed inside table of contents

### ISSUE THREAD

**Issue by ghost** [billryan/gitbook-plugin-etoc#2]
Hey, I I've recently discovered unexpected behavior of your plugin.

**Expected behavior:**
No toc is shown, and the content is not inside the toc.

**Actual behavior:**
![line](https://cloud.githubusercontent.com/assets/15706162/24119847/d6af85f0-0db2-11e7-9f1c-e7d50957ad6e.png)

**Workaround**
There seems to be a problem with the headings behavior. Using a single level 3 title or setting _mindepth_ to 2 fixes the problem. So I think there is something wrong with the headline detection. It will insert a empty toc that get filled with the actual site content.

**Setup:**
Node Docker Image 7.7.3
Gitbook 3.2.2
etoc 1.1.9

Custom css for dark theme:
```css
.book.color-theme-2 #toc, .book.color-theme-2.toc {
  background-color: rgba(255,255,255,.15);
}
```
Plugin is only activated -  no configuration is used 

Used markdown file:
[test.md.txt](https://github.com/billryan/gitbook-plugin-etoc/files/856249/test.md.txt)
(Renamed it because Github doesn't accept markdown files as an attachment)

**Comment by billryan** [comment]
Thanks for your such detailed bug report, I am a little bit busy this week and will fix it next weekend.

**Comment by billryan** [comment]
Fixed in 1.1.10. I should put the toc tag after heading depth check.
